elementary/utils/log.py

```diff
@@ -1,10 +1,11 @@
 import logging
+import os.path
 import sys
 
 from elementary.utils.env_vars import is_debug_mode_on
 
 FORMATTER = logging.Formatter("%(asctime)s — %(name)s — %(levelname)s — %(message)s")
-LOG_FILE = "edr.log"
+LOG_FILE = "logs/edr.log"
```

Comment:
I believe this should also use the `target-dir`, though I can see how this is a bit more challenging.

Reply:
Since logs exist also for dbt, I recommend it to be "logs" as well. Since we are logging while initializing the cli in elementary/cli/cli.py utilizing the taget_dir option on init is bit problematic. What do you suggest?

Reply:
@eran-levy @elongl what do you think about maybe adding the file handler to the logger only after we initialize the config object? (the logger can be initialized with only a console handler by default)

Reply:
@oravi Its possible but I think less recommended from design perspective as it will require that every new subcommand that will be added to remember and init the log file handler as well. Tight coupling both functionalities can be a bit problematic in my opinion.

Reply:
@eran-levy I agree that adding it to every subcommand is not good but maybe there's a way around it. Try to see how the CLI commands are structured and if you can maybe implement it in a single command

Reply:
ok checking

Reply:
@oravi I couldnt find an elegant way to abstract the creation of the logger and config while running subcommand - before moving forward so we wont write throw-away code, IMHO there are 2 options here:
1. Just adding a logger handler creation on every subcommand (in monitor command because I understood from @elongl lineage is going to be deprecated which means that some subcommands will have the functionality and some not) tightly coupled with the config and given arguments - I wont be able to add to all and if the user won't provide an argument we will need to store it in some default folder anyway.
2. Continue with the current approach - just persist it under "logs" (I think preferable as this is what dbt uses) or in another folder such as: "edr_output"

what do you suggest?